jobs/build-current-sqlite.groovy

```diff
@@ -33,33 +33,42 @@ that haven't yet been merged to master.""",
 ).apply();
 
 
+// This runs in webapp/.
```

Comment:
Thanks for this extra comment.  A lot of this stuff has abstractions that makes it not obvious what CWD is.